modules/gapi/include/opencv2/gapi/cpu/video.hpp

```diff
@@ -21,5 +28,66 @@ GAPI_EXPORTS GKernelPackage kernels();
 } // namespace gapi
 } // namespace cv
 
+#ifdef HAVE_OPENCV_VIDEO
+namespace cv { namespace gapi { namespace video {
+    struct BackgroundSubtractorParams
```

Comment:
Why are these parameters are under `HAVE_OPENCV_VIDEO` guard?

Please check that it really works: `cmake -DBUILD_opencv_video=OFF ...`
(it doesn't work, compilation of `opencv_test_gapi` is failed)